apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -205,10 +206,10 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
             return;
         }
 
-        const attributes = UserManagementConstants.SCIM2_ATTRIBUTES_DICTIONARY.get("ONETIME_PASSWORD");
+        const attributes: string = UserManagementConstants.SCIM2_ATTRIBUTES_DICTIONARY.get("ONETIME_PASSWORD");
 
         getUserDetails(user?.id, attributes)
-            .then((response) => {
+            .then((response: any) => {
```

Comment:
Can we use the right data type instead of `any`? Check the other places also.

Reply:
Let's avoid using `any`. `ProfileInfoInterface` is the correct type here.

Ref: https://github.com/wso2/identity-apps/blob/f9dde022fc4b18a19a503e5d1ec223926daf1904/apps/console/src/features/users/api/users.ts#L234

Reply:
Let's avoid the use of `any` wherever possible.


Ref: https://github.com/wso2/identity-apps/blob/f9dde022fc4b18a19a503e5d1ec223926daf1904/apps/console/src/features/users/api/users.ts#L234

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3643/commits/a21093f40a423fa6f6db8becd11bd586313a84ff.